Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Pointers/DefaultControllerPointer.prefab

```diff
@@ -158,7 +159,7 @@ LineRenderer:
       m_Curve:
       - serializedVersion: 3
         time: 0
-        value: 0.2
+        value: 0.15
```

Comment:
This looks like it's in the default controller pointer. Is that intentional, since this is a change to the line tether, not the line pointer?

Reply:
Intentional. Folks have complained about the default pointer being too wobbly since it was first introduced. I tweaked some values while I was adjusting the tether. Not worth a separate issue IMO.